### PR TITLE
New method: toString (223 B)

### PR DESCRIPTION
**Pull request by BabkinAleksandr**
[Ramda toString](http://ramdajs.com/docs/#toString)

`eslint-disable`, because of indent-error:

index_js_ ___documents_nanoutils

### CODE REVIEW

File: lib/toString/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      return 'new Date(' + JSON.stringify(value) + ')'
+    case '[object Object]':
+      var keys = Object.keys(value)
+      result = '{'
+      for (var i = 0; i < keys.length; i++) {
+        if (i !== 0) result += ', '
+        result += '"' + keys[i] + '"' + ': ' + toString(value[keys[i]])
+      }
+      result += '}'
+      return result
```
Comment (Beraliv) [inline lib/toString/index.js]: maybe `return result + '}'` without the line before it?

**Comment by Kelin2025** [comment]
Strings in `case` statements won't be minified. Maybe there is another way?  
Also, I think the first case should be `default`.

File: lib/toString/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    case '[object Function]':
+      return value + ''
+    case '[object Date]':
+      return 'new Date(' + JSON.stringify(value) + ')'
+    case '[object Object]':
+      var keys = Object.keys(value)
+      result = '{'
+      for (var i = 0; i < keys.length; i++) {
+        if (i !== 0) result += ', '
+        result += '"' + keys[i] + '"' + ': ' + toString(value[keys[i]])
```
Comment (Beraliv) [inline lib/toString/index.js]: Instead of `'"' + ': '` use `'": '`, at least it's shorter

File: lib/toString/index.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      }
+      result += '}'
+      return result
+    case '[object Array]':
```
Comment (Beraliv) [inline lib/toString/index.js]: you probably forgot about `[object Arguments]`

**Comment by BabkinAleksandr** [comment]
Done! Added `default` and `arguments` support

Also, I've tried to reuse loop like this:
```js
case '[object Arguments]':
case '[object Array]':
  for (var i = 0; i < value.length; i++) {
    if (i !== 0) result += ', '
      result += toString(value[i])
    }
  }
  return Array.isArray(value)
    ? '[' + result + ']'
    :  '(function() { return arguments; }(' + result + '))'
```
...but it increases method's size for 8 Bytes and in theory could be a bit slower.

File: lib/toString/index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      for (var i = 0; i < value.length; i++) {
+        if (i !== 0) result += ', '
+        result += toString(value[i])
+      }
+      return '(function() { return arguments; }(' + result + '))'
+    case '[object Array]':
+      for (var i = 0; i < value.length; i++) {
+        if (i !== 0) result += ', '
+        result += toString(value[i])
+      }
```
Comment (Beraliv) [inline lib/toString/index.js]: you repeat exactly the same code, maybe `function` for this?

**Pull request merged**